lib/billy/rspec.rb

```diff
@@ -1,20 +1,8 @@
+# Deprecated
 require 'capybara/rspec'
-require 'billy'
+require 'billy/browsers/capybara'
+require 'billy/init/rspec'
 
-Billy.register_drivers
+Billy::Browsers::Capybara.register_drivers
```

Comment:
Can we just `require billy/capybara/rspec` here instead of having this and above duplicated in two places?  Question applies to all of the deprecated files.